docs/modules/ROOT/partials/getting-started/con-registry-logging.adoc

```diff
@@ -0,0 +1,47 @@
+[id="registry-logging"]
+= {registry} Logging
+
+{registry} logging configuration can be modified while {registry} it's running. There is a REST endpoint that allows to configure the log level for specific loggers, allowing for finer grained logging configurations.
```

Comment:
Suggest sentence style heading and some minor copy edits:
= {registry} logging

You can update {registry} logging configuration at runtime. {registry} provides a REST endpoint to set the log level for specific loggers for finer grained logging.